langstream-cli/src/main/java/ai/langstream/cli/commands/applications/UIAppCmd.java

```diff
@@ -58,17 +61,20 @@ public class UIAppCmd extends BaseApplicationCmd {
     private String applicationId;
 
     @CommandLine.Option(
-            names = {"-p", "--port"},
-            description = "Port for the local webserver and UI. If 0, a random port will be used. ",
-            defaultValue = "8092")
+        names = {
```

Comment:
this file has been reformatted

please run this command to make it complaint with the code style

mvn generate-sources

Reply:
I spent some time trying to figure what style to use.  Got educated on java style. thx will do.  BTW, I do everything at the command-line.  I use to use jetbrains -probably setup it up at some point.

Reply:
done